allofplos/article_class.py

```diff
@@ -201,7 +201,10 @@ def get_element_xpath(self, tag_path_elements=None):
                                  'custom-meta',
                                  'meta-value')
         tag_location = '/'.join(tag_path_elements)
-        return self.root.xpath(tag_location)
+        if remote is False:
```

Comment:
You can simplify the return logic by using this conditional to define a local variable root and then have a single return statement.

E.g., 

```python
if not remote:
    root = self.root
else:
    root = self.remote_tree.getroot()
return root.xpath(tag_location)
```

Reply:
brilliant thanks! fixed in https://github.com/PLOS/allofplos/pull/54/commits/eddfe6545b386eac9e725d3d64bc9b769677d08a

Reply:
Also I'd really do it as the reverse:
```python
if remote:
    root = self.remote_tree.getroot()
else:
    root = self.root
return root.xpath(tag_location)
```

Reply:
Aaaaand I just noticed that was how you actually implemented this 👍  well done.